api/bin/cli.ts

```diff
@@ -2,17 +2,17 @@ import { start } from 'repl';
 
 import { Connection } from 'typeorm';
 
-import { createDatabaseConnection, UserRepo } from '../src/db';
-import { User } from '../src/db/entities/user';
+import { createDatabaseConnection, Repos } from '../src/db';
+import * as Entities from '../src/db/entities';
 
 createDatabaseConnection().then((connection: Connection) => {
   const context = start('$ ').context;
 
   Object.assign(context, {
     connection,
     manager: connection.manager,
-    repos: { UserRepo },
-    User,
+    repos: { ...Repos },
```

Comment:
it is not work like you think it's work.
before it was strongly typed and I get good hints in CLI.
e.g. I could write:
`repos.UserRepo.reg`, press tab and get `register` word.
now it is not work. you spead an array inside a object (I not even sure how it works, probably create an object with 0, 1, 2 indexes).

please, restore old version of the code, old get with solution that properly support typings.